hc-bar/js/bar.js

```diff
@@ -0,0 +1,314 @@
+// series is an array of objects
+//    name - eventType (Intercept, Strike, Other)
+//    data is an array of objects
+//      name - monthYear
+//      Y - # of event in m/y
+//      drilldown - monthYear-eventType
+
+// drilldown is an object
+//    allowPointDrilldown - false (drilldown shows events for all categories in that month/year)
+//    series is an array of objects
+//      name - eventType+Headline+Text (Strike, Intercept, Other)
+//      id - monthYear-eventType
+//      data is an array of arrays
+//        [day, # of events]
+
+var drilldownSeries = [];
+// Create empty array for new event objects
+var series = [];
+
+Highcharts.data({
+  // Load Data in from Google Sheets
+  googleSpreadsheetKey: "1AKM59m3iaOSSQgVsIoDcSWxjcpX781JIuBwB9YdDgFs",
+  googleSpreadsheetWorksheet: 1,
+  switchRowsAndColumns: true,
+  parsed: function parsed(columns) {
+    // set default values
+    var drilldown = "";
+    var dataObject = { other: {}, strike: {}, intercept: {} };
+    var dataArray = [];
+    var drilldownObject = {};
+
+    columns.forEach(function (row, index) {
+      // Skip header row and title row
+      if (index == 0 || index == 1) {
+        return;
```

Comment:
In JS if you are checking for an exact comparison, it is best to use a triple equals (===). This is because it checks for both the type and the value.